.github/pull_request_template.md

```diff
@@ -59,6 +60,9 @@ Operator (before pushing merge the commit)
 
 Operator (after pushing the merge commit)
 
+<!-- Announcements --> 
+
+- [ ] Made requested announcement from author               <sub>or author does not request an announcement</sub>
```

Comment:
Made announcement requested by author or description does not contain an announcement